base/common/python/pki/cert.py

```diff
@@ -98,6 +98,7 @@ class CertDataInfo(object):
         'id': 'serial_number', 'SubjectDN': 'subject_dn', 'Status': 'status',
         'Type': 'type', 'Version': 'version', 'KeyLength': 'key_length',
         'KeyAlgorithmOID': 'key_algorithm_oid',
+        'IssuerDN': 'issuer_dn',
```

Comment:
It is not necessary but the attribute could be initialised in the `init` method below for consistency with the other